services/net/contentmigration/migrators/ContentMigrator.cs

```diff
@@ -177,5 +179,54 @@ public ContentReferenceModel CreateContentReference(SourceModel source, string t
         return product;
     }
 
+    /// <summary>
+    /// map possible Action values to Actions
+    /// </summary>
+    /// <param name="frontPageStory"></param>
+    /// <param name="wapTopStory"></param>
+    /// <param name="alert"></param>
+    /// <param name="commentary"></param>
+    /// <param name="commentaryTimeout"></param>
+    /// <returns></returns>
+    public IEnumerable<Kafka.Models.Action> GetActionMappings(bool frontPageStory, bool wapTopStory, bool alert, bool commentary, double? commentaryTimeout) {
+        List<Kafka.Models.Action> mappedActions = new();
+        string actionName;
+        ActionType actionType;
+
+        if (frontPageStory) {
```

Comment:
Looks like possible duplication that could be reduced.  Not important for this work however.